httpservice/Dockerfile

```diff
@@ -1,8 +1,13 @@
 FROM python:3.11-slim as tippecanoe-build-image
 
 RUN apt-get update && apt-get install -y --no-install-recommends build-essential libsqlite3-dev zlib1g-dev git
-RUN git clone https://github.com/mapbox/tippecanoe.git && cd tippecanoe && make -j && make install && cd .. && rm -r tippecanoe
-
+RUN git clone https://github.com/felt/tippecanoe.git && cd tippecanoe && make -j && make install && cd .. && rm -r tippecanoe
+RUN apt-get install -y binutils libproj-dev gdal-bin
```

Comment:
This line may need to go in the second image below, and not here.

The image we are in now is just a temporary one that we're going to scavenge binaries out of in the image we'll actually deploy below.  I imagine that one we deploy needs to have gdal.

Reply:
Ah, yeah. Oversight; thanks.